adagrams/game.py

```diff
@@ -1,11 +1,107 @@
+
+# import random
+import random
+
+import copy 
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+letter_list = []
+for letter in LETTER_POOL.keys():
+    letter_list += LETTER_POOL[letter] * letter
+# print(letter_list)
```

Comment:
It’s good practice to avoid committing commented out code when submitting a PR.

Reply:
A better place for this code would be inside the `draw_letters()` function since we're only using the `letter_list` in this function. It should live there 👍🏾